tokio-test/src/io.rs

```diff
@@ -200,9 +203,11 @@ impl Inner {
     }
 
     fn poll_action(&mut self, cx: &mut task::Context<'_>) -> Poll<Option<Action>> {
-        use futures_core::stream::Stream;
-
-        Pin::new(&mut self.rx).poll_next(cx)
+        if let Some(fut) = &mut self.rx_fut {
+            Pin::new(&mut *fut).poll(cx)
```

Comment:
We *could* just expose `poll_recv()` :)

Reply:
MEH!

Reply:
replaced this with `async-stream` version.